eos/parameters/sm-and-eft.yaml

```diff
@@ -1322,7 +1322,7 @@ groups:
         The list of parameters describing the $\bar{s}b\bar{s}b$ Wilson coefficients in the effective Hamiltonian
         \begin{equation}
           \mathcal{H}^{\bar{s}b\bar{\nu}\nu}_\mathrm{eff}
-          = \frac{4 G_F}{\sqrt{2}} V_{tb}^* V_{ts} \sum_i \mathcal{C}^{\bar{s}b\bar{\nu}\nu}_i \mathcal{O}^{\bar{s}b\bar{\nu}\nu}_i\,.
+          = \frac{4 G_F}{\sqrt{2}} \frac{\alpha_e}{2 \pi} V_{tb}^* V_{ts} \sum_i \mathcal{C}^{\bar{s}b\bar{\nu}\nu}_i \mathcal{O}^{\bar{s}b\bar{\nu}\nu}_i\,.
```

Comment:
Is this in agreement with the current implementation?

Reply:
Yes, it is then consistent with 
https://github.com/eos/eos/blob/master/eos/models/wilson-coefficients.hh#L174
and the implementation here
https://github.com/eos/eos/blob/master/eos/b-decays/b-to-psd-nu-nu.cc#L192